axiomapy/examples/with_axiomaapi.ipynb

```diff
@@ -363,11 +363,48 @@
    "source": [
     "results = AnalysesRiskAPI.get_risk_model_results(request_id=requestId, show_raw_results=False)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The risk model data from the above step can also fetched in zipped format. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "headers={\"Accept\":\"multipart/x-zip\"}\n",
+    "raw_results = AnalysesRiskAPI.get_risk_model_results(request_id=requestId, headers=headers)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This data can be extracted and saved to a local directory \"C:/RMS/temp\" with a sample code below"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import zipfile\n",
+    "from io import BytesIO\n",
+    "z = zipfile.ZipFile(BytesIO(raw_results.content))\n",
+    "for name in z.namelist():\n",
+    "    z.extract(name, \"C:/RMS/temp/\")"
```

Comment:
should a relative path be used?

Reply:
I agree with Weiwei here - we shouldn't pass a path. The default arg is None and it will then extract to the users current directory.

Reply:
Will be happy to change it.
My thought process was if users come up asking can we add a dir, however since this code is outside the scope of axioma-py it should not matter!